runners/spark/src/main/java/org/apache/beam/runners/spark/io/ConsoleIO.java

```diff
@@ -21,32 +21,31 @@
 import org.apache.beam.sdk.values.PCollection;
 import org.apache.beam.sdk.values.PDone;
 
-/**
- * Print to console.
- */
+/** Print to console. */
 public final class ConsoleIO {
 
-  private ConsoleIO() {
-  }
+  private ConsoleIO() {}
 
-  /**
-   * Write on the console.
-   */
+  /** Write to console. */
   public static final class Write {
 
-    private Write() {
-    }
+    private static final int DEFAULT_NUM = 10;
+
+    private Write() {}
 
+    /** Prints {@value #DEFAULT_NUM} elements from the {@link PCollection} to stdout. */
     public static <T> Unbound<T> out() {
-      return new Unbound<>(10);
+      return new Unbound<>(DEFAULT_NUM);
     }
 
+    /** Prints {@code num} elements from the {@link PCollection} to stdout. */
     public static <T> Unbound<T> out(int num) {
       return new Unbound<>(num);
     }
 
     /**
      * {@link PTransform} writing {@link PCollection} on the console.
+     *
```

Comment:
Please use consistent terminology.
Some of javadoc says `Print to console`, some says `Print to stdout` and some says `writing to stdout`.

I suggest you pick one term and stick with it (either `write` or `print`, either `stdout` or `console`).

Reply:
I've changed all of those to `console`, and left the original discrepancy of `\*Print to console*\` and `\*Write to console*\`.